### PR TITLE
Composerification

### PR DESCRIPTION
**Pull request by tobybellwood**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "govcms/dvf",
```
Comment (jorgegc) [inline composer.json:2]: Why are we namespacing it under `govcms` and not `drupal`?

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    {
+      "name": "Toby Bellwood",
+      "homepage": "https://www.drupal.org/u/tobybellwood"
+    }
+  ],
+  "support": {
+    "issues": "https://www.drupal.org/project/issues/dvf"
+  },
+  "license": "GPL-2.0+",
+  "version": "0.0.1",
```
Comment (jorgegc) [inline composer.json:20]: Version is not required.

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "minimum-stability": "alpha",
+  "prefer-stable": true,
```
Comment (jorgegc) [inline composer.json:22]: This is not going to have any effect because `prefer-stable` is root-only.

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  "require": { }
```
Comment (jorgegc) [inline composer.json:23]: Empty `require` should be removed.

**Comment by jorgegc** [comment]
Hi @tobybellwood there are a number of issues with this PR, including conflicts. This is blocking #4 so I think it is best if we close this one and incorporate these changes in my next PR. Thoughts?

**Review by jorgegc — CHANGES_REQUESTED** [review]

**Comment by tobybellwood** [comment]
Happy to close and have a resubmit

**Pull request closed without merging**